diff --git a/ios/StripeSwiftInterop.h b/ios/StripeSwiftInterop.h
index d9ca956..ce4fe3f 100644
--- a/ios/StripeSwiftInterop.h
+++ b/ios/StripeSwiftInterop.h
@@ -11,7 +11,7 @@
 @protocol STPPaymentCardTextFieldDelegate;
 @protocol STPCardFormViewDelegate;
 
-typedef NS_ENUM(NSUInteger, STPPaymentStatus);
+typedef NS_ENUM(NSInteger, STPPaymentStatus);
 
 // stripe_react_native-Swift.h also depends on these headers.
 #import <React/RCTBridgeModule.h>
